src/navigation/docs.js

```diff
@@ -87,6 +87,10 @@ export const navigation = [
                 title: 'Accessing the Internet',
                 href: '/docs/creators/javascript/guides/accessing-internet',
               },
+              {
+                title: 'VM Filesystem',
```

Comment:
Additional comment not sure where to place:

I would recommend revisiting the whole article and create some proper headings as they are used a lot during search. If you don't have the headings then people need to actively find the article because it's not weighted high during search. 

When constructing the headings think of keywords. I would image especially things like `send failed because receiver is gone` is a common error to experience when working with golem and is caused by trying to transfer to/from a path not defined as a VOLUME